crates/rome_js_analyze/src/analyzers/nursery/use_exponentiation_operator.rs

```diff
@@ -0,0 +1,303 @@
+                JsAnyExpression::from(new_node),
+            );
+        }
+
+        Some(JsRuleAction {
+            category: ActionCategory::QuickFix,
+            applicability: Applicability::MaybeIncorrect,
+            message: markup! { "Use the '**' operator instead of 'Math.pow'." }.to_owned(),
+            mutation,
+        })
+    }
+}
+
+/// Verify if the autofix is safe to be applied and won't remove comments.
+/// Argument list is considered valid if its length is 2 and there's no spread arg.
+fn should_suggest_fix(node: &JsCallExpression) -> Option<bool> {
+    let arguments = node.arguments().ok()?;
+    let args_count = arguments.args().len();
+
+    Some(
+        args_count == 2
+            && !arguments.l_paren_token().ok()?.has_leading_comments()
+            && !arguments.l_paren_token().ok()?.has_trailing_comments()
+            && !arguments.r_paren_token().ok()?.has_leading_comments()
+            && !arguments.r_paren_token().ok()?.has_trailing_comments()
+            && arguments
+                .args()
+                .into_iter()
+                .filter_map(|arg| arg.ok())
+                .all(|arg| {
+                    !arg.syntax().has_leading_comments()
+                        && !arg.syntax().has_trailing_comments()
+                        && arg.as_js_spread().is_none()
+                }),
+    )
+}
+
+/// Wraps a [JsAnyExpression] in paretheses
+fn parenthesize_js_any_expression(expr: &JsAnyExpression) -> JsAnyExpression {
+    JsAnyExpression::from(make::js_parenthesized_expression(
+        make::token(T!['(']),
+        expr.clone(),
+        make::token(T![')']),
+    ))
+}
+
+/// Determines whether the given node needs parens if used as the base in an exponentiation binary expression.
+fn does_base_need_parens(expr: &JsAnyExpression) -> Option<bool> {
+    Some(
+        // '**' is right-associative, parens are needed when Math.pow(a ** b, c) is converted to (a ** b) ** c
+        expr.precedence().ok()? <= OperatorPrecedence::Exponential
+			// An unary operator cannot be used immediately before an exponentiation expression
+            || expr.as_js_unary_expression().is_some()
+            || expr.as_js_await_expression().is_some(),
+    )
+}
+
+/// Determines whether the given node needs parens if used as the exponent in an exponentiation binary expression.
+fn does_exponent_need_parens(expr: &JsAnyExpression) -> Option<bool> {
+    Some(expr.precedence().ok()? < OperatorPrecedence::Exponential)
+}
+
+/// Determines whether the given parent node needs parens if used as the exponent in an exponentiation binary expression.
+fn does_parent_expression_need_parens(node: &JsCallExpression) -> Option<JsAnyExpression> {
+    // Skips already parenthesized expressions
+    if has_parenthesized_parent_expression(JsAnyExpression::from(node.clone()))? {
+        return None;
+    }
+
+    let parent = node.parent::<JsAnyExpression>()?;
+
+    let needs_parentheses = match parent.clone() {
```

Comment:
It should be possible to use `&parent` as you aren't consuming the `parent` in any of the branches. 
```suggestion
    let needs_parentheses = match &parent {
```